source/retryable-reads/tests/listCollectionNames.yml

```
@@ -0,0 +1,59 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data: []
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "ListCollectionNames succeeds on first attempt"
+        clientOptions:
+            retryReads: true
+        operation: &retryable_operation
+            name: listCollectionNames
+            object: database
+        expectations:
+            -  &retryable_command_started_event
+                command_started_event:
+                    command:
+                        listCollections: 1
+    -
+        description: "ListCollectionNames succeeds on second attempt"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands:
+                        - listCollections
+                closeConnection: true
+        operation: *retryable_operation                
+        expectations:
+             - *retryable_command_started_event
```

Missing the second command:
```suggestion
             - *retryable_command_started_event
             - *retryable_command_started_event
```